src/Polly.Core/Retry/RetryHelper.cs

```diff
@@ -31,6 +31,11 @@ internal static class RetryHelper
         }
         catch (OverflowException)
         {
+            if (maxDelay is not null)
+            {
+                return maxDelay.Value;
```

Comment:
```suggestion
            if (maxDelay is {} value)
            {
                return value;
```

Reply:
Just for my understanding, why do you prefer `is {} value` to `is not null`?

Reply:
Value will re-check for null internally (so it could throw if it wasn't), where-as doing it this way means it only checks once and we have access to the value when it isn't.

Reply:
I've commited your suggestion, thanks for the explanation.

Reply:
I've commited your suggestion, thanks for the explanation.